lib/app/service/app.go

```diff
@@ -78,8 +78,6 @@ type Config struct {
 	CacheResources bool
 	// UnpackedDir is the dir where packages are unpacked
 	UnpackedDir string
-	// ExcludeDeps defines a list of dependencies that will be excluded for the app image
-	ExcludeDeps []loc.Locator
```

Comment:
I had to remove this since the implementation of filtering of the excluded applications is not complete - e.g. it does not work for the cluster application service running inside of the gravity-site Pod.
To counter this, I removed this configuration and instead compute the list of excluded applications each time it is required (see below).